ansible/roles/vm_set/tasks/configure_server.yml

```diff
@@ -0,0 +1,234 @@
+- name: Check flag file to see if server is configured
```

Comment:
Maybe we need a parameter to forcely clear this flag in some scenarios?
The parameter could be added separately in another PR.

Reply:
Path of the flag file should be a hard coded constant. It's like a protocol that all server configuration tasks should honor. It is not something that supposed to be provided through parameter during runtime.